app/controllers/pages_controller.rb

```diff
@@ -98,6 +98,33 @@ def common_params
     }
   end
 
+  def multiupload_files_remaining
+    max_files = @page.components.select { |c| c.type == 'multiupload' }.first['max_files'].to_i
+    answered = false
```

Comment:
can you help me understand what's happening here? I don't fully understand why we are assigning this variable to a boolean and then using it in the if/else statements

Reply:
It wasn't actually being used, it just needed to exist for the template to work on the edit page. Have removed!